Events/BodyScannedEvent.cs

```diff
@@ -136,5 +140,84 @@ private decimal sanitiseCP(decimal cp)
                 return Math.Round(cp * 100);
             }
         }
+
+        private long? estimatevalue(bool? dssequipped)
+        {
+            int bodydataconstant = 720;
+            int terraconstant = 0;
+            bool terraformable = false;
+            double value;
+            double basevalue;
+            double terrabonusvalue;
+
+            // Override constants for specific types of bodies
+            if ( (terraformstate == "Terraformable") || (terraformstate == "Candidate for terraforming") )
+            {
+                terraformable = true;
+            }
+            if ( bodyclass == "Ammonia world")
+            {
+                // Ammonia worlds
+                bodydataconstant = 232619;
```

Comment:
These magic numbers are only likely to be used in this location. I suppose the best reason to do this might be to make sure that the constants can be updated independent of the code (for example, if the json was online).